src/h5web/visualizations/heatmap/DomainSlider.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import ReactSlider from 'react-slider';
+import shallow from 'zustand/shallow';
+import { format } from 'd3-format';
+import styles from './DomainSlider.module.css';
+import { useHeatmapStore, Domain } from './store';
+
+const EXTEND_PERCENTAGE = 0.2;
+const MIN_DECIMALS = 1;
+
+function round(value: number, decimals: number): number {
+  return Number(`${Math.round(Number(`${value}e${decimals}`))}e-${decimals}`);
+}
```

Comment:
An function to round accurately numbers based on https://www.jacklmoore.com/notes/rounding-in-javascript/. May be moved in `utils` or replaced by https://lodash.com/docs/4.17.15#round.

Reply:
We already have `lodash-es`, so feel free to import `{ round }` from it ;)